deploy/kubernetes/elastic-agent-kustomize/default/elastic-agent-standalone/base/elastic-agent-standalone-daemonset-configmap.yaml

```diff
@@ -359,28 +359,26 @@ data:
               #     match: after
             paths:
               - /var/log/containers/*${kubernetes.container.id}.log
-      - name: filestream-generic
```

Comment:
Those should also be remove completely with https://github.com/elastic/ingest-docs/pull/304.

Reply:
Removed!